src/NuGet.Clients/PackageManagement.UI/Xamls/OptionsControl.xaml

```diff
@@ -32,8 +35,18 @@
         <RowDefinition Height="auto" />
         <RowDefinition Height="auto" />
         <RowDefinition Height="auto" />
+        <RowDefinition Height="auto" />
       </Grid.RowDefinitions>
 
+      <Grid Visibility="{Binding IsSolution, Converter={StaticResource BooleanToVisibilityConverter}}" Grid.Row="0" Name = "warningTextGrid" Margin="2,3,2,3">
+        <Grid.ColumnDefinitions>
+          <ColumnDefinition Width="Auto" />
+          <ColumnDefinition Width="*"/>
+        </Grid.ColumnDefinitions>
+        <Image Grid.Row="0" Grid.Column = "0" Name = "WarningTextImage"  Height="16" Width="16" Source="{StaticResource BitmapImage_StatusInformation}"/>
```

Comment:
no need to give a name, until you need to refer to it at another place in XAML or in code-behind.